basyx.aasrepository/basyx.aasrepository-backend-inmemory/src/test/java/org/eclipse/digitaltwin/basyx/aasrepository/TestInMemoryAasRepository.java

```diff
@@ -31,7 +31,8 @@
 import org.junit.Test;
 
 /**
- * 
+ * Testing the InMemoryAasRepository's name configured through the constructor
+ *  
```

Comment:
Please change it to "Tests the @link InMemoryAasRepository name"

Reply:
done